platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gui/internal/WinTabAddresses.java

```diff
@@ -52,13 +53,15 @@ public void prePaint() {
         String s = "";
         synchronized (getPlatforms()) {
             for (final Platform p : getPlatforms()) {
```

Comment:
Minor: this is a very poor anti-pattern. If the getter ever becomes dynamic and could return different objects, this synchronization scheme will break apart. A local variable is needed here. This may never become a problem in this particular code, but an engineer seeing this code and then replicating this pattern elsewhere could run into a big trouble...

Reply:
Filed https://github.com/hashgraph/hedera-services/issues/16726 .